### ISSUE TITLE
Changing the SPI pinouts

### ISSUE THREAD

**Issue by ravipurigen** [wrobell/fpulse#4]
Is it possible to change the SPI pinouts defined in tlc5947.py to use the following instead ? Seems like it is hardcoded in the bmc2835_spi_begin() function defined in bcm2835.c
GPIO38 - SPI0_MOSI
GPIO39 - SPI0_SCLK
GPIO36 - SPI0_CE0_N

**Comment by wrobell** [comment]
Is this for Raspberry Pi?

**Comment by ravipurigen** [comment]
Yes, the bcm2835 library seems to have hardcoded the SPI pins to use 7-11 in, so I am guessing I will have to rebuild that one and there's no modification needed on the fpulse side, right ?

**Comment by wrobell** [comment]
Is it possible to assign different GPIOs than the default one to SPI in Raspberry Pi?

**Comment by ravipurigen** [comment]
Yes, line number 1273 at https://github.com/raspberrypi/firmware/blob/master/boot/overlays/README
I ran the spidev_test.c to confirm it and the default SPI pins gets disabled after adding that line to the /boot/config.txt
